internal/multirun.py

```diff
@@ -89,13 +89,15 @@ def _script_path(workspace_name: str, path: str) -> str:
     else:
         return _R.Rlocation(f"{workspace_name}/{path}")
 
+
 def _main(path: str) -> None:
     with open(path) as f:
         instructions = json.load(f)
 
     workspace_name = instructions["workspace_name"]
     commands = [
-        Command(_script_path(workspace_name, blob["path"]), blob["tag"], blob["args"], blob["env"])
+        Command(_script_path(workspace_name, blob["path"]), blob["tag"],
+                blob["args"] + (sys.argv[2:] if len(sys.argv) > 2 else []), blob["env"])
```

Comment:
I think this is a reasonable default behavior, and it's what I want for rules_lint.

I suspect users are going to file feature requests to pass a subset of flags to different runnables though. Might be good to decide now if you're open to that or want to keep it simple.